### PR TITLE
Changes made for CISTUP task

### PR DESCRIPTION
**Pull request by deepalics0044**
changes made

### CODE REVIEW

**Comment by shankari** [comment]
I haven't been checking in `package-lock.json`, can you remove it?
I'm happy to merge the other changes if it turns out that is the text required.
Did you check with Tarun or just make the changes yourself?

**Comment by shankari** [comment]
@deepalics0044 did you get confirmation from Tarun/Ipsita/Abdul that this is the text that they want?
Because I can see a bunch of places where the text is inaccurate.

File: www/templates/intro/consent.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 <div class="consent-text">
 <h3>Introduction and Purpose</h3>
 This app is part of a travel behavior research platform developed at the
-University of California, Berkeley. The platform development is being led by <a
+Indian Institute of Science, Bengaluru. The platform development is being led by <a
```
Comment (shankari) [inline www/templates/intro/consent.html]: The platform was still developed at UC Berkeley. IISc is mainly customizing it. You can't say that the platform is being developed at IISc and then say that I am leading the development! I am a UCB grad student!

File: www/templates/intro/login.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 <div class="intro-title">
-<center><h3>Login via google</h3></center></div>
+<center><h3>Login using a google account</h3></center></div>
 
 <div class="intro-text">
 Currently, we only support logging in via google, since they support techniques
-such as two factor authentication for greater security. Participants at UC
-Berkeley can choose to login using either their CalNet ID or a personal gmail
+such as two factor authentication for greater security. Participants at 
+IISc can choose to login using either their iiscwlan or a personal gmail
```
Comment (shankari) [inline www/templates/intro/login.html]: participants at IISc can't login using their iiscwlan; you have not yet integrated with the iisc authentication system, whatever it is. They can only login using personal gmail addresses.

File: www/templates/intro/summary.html [hunk 3]
```diff
@@ -9,7 +9,7 @@
 	<div class="intro-space"></div>
 	<ol>
 	<li> <b>Automatically calculate</b> your transportation carbon footprint and
-	compare it against California targets
+	compare it against Bangalore targets
```
Comment (shankari) [inline www/templates/intro/summary.html]: There are Bangalore targets?

**Comment by shankari** [comment]
Also, what is the difference between this and #449? Why do you have duplicate pull requests?

**Comment by deepalics0044** [comment]
Okay I just made the changes to see whether it reflects on the phone or
not. It was not the final changes.I beg your pardon.

**Comment by shankari** [comment]
No worries - you can only learn by doing! I encourage you to make changes and see them reflected in the UI - I just don't want the resulting changes to be more confusing than the original.

How about you revert the changes to consent and summary and for the login, say only that people can sign in with their personal email address (i.e. remove the part about sign in with IISc wlan)?

I will merge that and deploy tonight, you can test it, and send out email to Tarun and Ranju tomorrow...

**Comment by shankari** [comment]
Ah, I see that #449 is to the correct branch.
I will merge that instead and close this.
But since this is more recent than #449, I still encourage you to be careful while creating pull requests in the future.

**Pull request closed without merging**